### ISSUE TITLE
Who owns conjars.org? Unable to register

### ISSUE THREAD

**Issue by JLLeitschuh** [clojars/clojars-web#741]
Hi,

This project seems to have been the historical predecessor to https://conjars.org
Is there any idea who owns that project? It seems like it's impossible to register an account to it (attempting seems to fail weirdly).

It looks pretty ancient, but it looks like it may be used still by several open source projects as an artifact repository. I'm concerned about it's security, but I can't even create an account against it.

**Comment by tobias** [comment]
@cwensel is the maintainer of conjars.org, but I'm not sure where the source for his clojars fork lives.

**Comment by cwensel** [comment]
hey @JLLeitschuh 

yep, that was me. I don’t own those systems anymore. and yes, the server and code is ancient. 

Account creation is failing (intentionally), but the server is live so builds and links don’t break.

I’ve been trying to find a way to swap this out with a mirror (via s3 etc) as the owners don’t have any intentions to maintain the systems. I also don’t own the domain name, so any efforts may be thwarted.

Ping me offline if you have any specific question or concerns.

**Comment by JLLeitschuh** [comment]
@cwensel I've sent you an email.

TL;DR: I'm looking to prevent a repeat of this incident:
https://blog.autsoft.hu/a-confusing-dependency/

**Comment by tobias** [comment]
Thanks @cwensel, @JLLeitschuh. I'm closing this issue.
